src/Traversal/Sdk/Traversal.targets

```
@@ -26,9 +26,10 @@
     <EnableDefaultItems>false</EnableDefaultItems>
 
     <!--
-      TargetFramework is required for restore and used to default to .NET Framework v4.5.  However, Traversal projects don't specify a version so it needs to be defaulted here.
+      TargetFramework is required for restore. If the user specifies a version, use that. Otherwise, use the
+      NETCoreAppMaximumVersion so that the project acts as if it were part of the SDK.
     -->
-    <TargetFramework Condition="'$(TargetFramework)' == ''">net45</TargetFramework>
+    <TargetFramework Condition=" '$(TargetFramework)' == '' ">net$(NETCoreAppMaximumVersion)</TargetFramework>
```

Just in case `NETCoreAppMaximumVersion` isn't defined.

```suggestion
    <TargetFramework Condition=" '$(TargetFramework)' == '' And '$(NETCoreAppMaximumVersion)' != ''">net$(NETCoreAppMaximumVersion)</TargetFramework>
    <TargetFramework Condition=" '$(TargetFramework)' == ''">net45</TargetFramework>
```

---

I originally had that, then convinced myself that it could never be unset 😅.

Added, but for my education, do you know of any circumstances where it wouldn't be set?

---

Shouldn't it always be set at this point because it's set in the 'base' SDK's Microsoft.NETCoreSdk.BundledVersions.props file, which is included by the 'base' SDK's props pass, which is imported by this SDK already?

---

And it is set in .props and we're reading in .targets, so it's _technically possible_...